packages/gogs/build.sh

```diff
@@ -3,10 +3,20 @@ TERMUX_PKG_DESCRIPTION="A painless self-hosted Git service."
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="Injamul Mohammad Mollah <mrinjamul@gmail.com>"
 TERMUX_PKG_VERSION=0.12.3
+TERMUX_PKG_REVISION=2
 TERMUX_PKG_SRCURL=https://github.com/gogs/gogs/archive/v$TERMUX_PKG_VERSION.tar.gz
 TERMUX_PKG_SHA256=6a0e1e369d2e097a73fe99fb78046db0b022ce0c229d3977068e7b21e2e364c8
 TERMUX_PKG_DEPENDS="dash, git"
 TERMUX_PKG_CONFFILES="etc/gogs/app.ini"
+TERMUX_PKG_HOSTBUILD=true
+
+termux_step_handle_hostbuild() {
+	termux_setup_golang
+	export GOPATH=$TERMUX_PKG_HOSTBUILD_DIR
+	mkdir -p $TERMUX_PKG_HOSTBUILD_DIR
+	cd $TERMUX_PKG_HOSTBUILD_DIR
+	go get -u github.com/kevinburke/go-bindata/...
```

Comment:
Which program is it that is needed? (Maybe it can be installed through apt in ubuntu instead?)

Reply:
It require go-bindata, the version from kevinburke only as stated in their doc